### PR TITLE
feat: link testimonial profiles

### PR DESCRIPTION
**Pull request by adityassharma-ss**
## Fixes Issue: 

Closes [FEATURE] Linked testimonial profiles #2444 

## Changes proposed

I have updated the `testimonial.js` file that adds a new feature solving the ISSUE #2444 .

It follows all the listed testcases
## Check List (Check all the applicable boxes) 

- [ X] My code follows the code style of this project.
- [ X] My change requires changes to the documentation.
- [ X ] I have updated the documentation accordingly.
- [ X] All new and existing tests passed.
- [ X] This PR does not contain plagiarized content.
- [ X] The title of my pull request is a short description of the requested changes.

## Screenshots

Eddie Jaoude - Brave 12_21_2022 4_13_21 PM

## Note to reviewers
This adds a feature to the existing code content & it has passed all the testcases. The feature will work easily after merging.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2446"></a>

### CODE REVIEW

File: components/user/UserTestimonials.js [hunk 1]
```diff
@@ -51,4 +53,4 @@ export default function UserTestimonials({ users, data, BASE_URL }) {
         ))}
     </>
   );
-}
+}
```
Comment (schmelto) [inline components/user/UserTestimonials.js:56]: There is a missing endline could you add this one?

File: data/adityassharma-ss.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
```
Comment (schmelto) [inline data/adityassharma-ss.json:1]: Its best to separate two different additions into two different prs could you remove the profile addition from this pr and include it into another one?

File: package-lock.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "1.12.0",
+      "version": "1.24.9",
```
Comment (schmelto) [inline package-lock.json:9]: We normally do not commit changes into `package-lock.json`. Could you revert the changes made in this file?

**Review by schmelto — CHANGES_REQUESTED** [review]
Some comments.

If you need any assistance please let us know.

File: data/adityassharma-ss.json (hunk 2 above)
Comment (schmelto) [reply data/adityassharma-ss.json:1]: ## Smaller and focused Pull Requests (PR)

### TLDR;

The smaller and more focused the PR the easier for the reviewer and the more likely it is to to merge faster.

---

### Why are smaller PRs better?

- Easier for the reviewer
- Will get merged faster
- Reduces conflicts with other changes
- You get more green squares if you separate PRs out into focused changes

---
**Summary:** :hammer_and_wrench: This code needs a fix

File: package-lock.json (hunk 3 above)
Comment (adityassharma-ss) [reply package-lock.json:9]: Yeah. Sure. Sorry.

**Comment by eddiejaoude** [comment]
@adityassharma-ss we just released new documentation that will help, any questions let us know https://linkfree.eddiehub.io/docs/quickstart
